epinorm/norm.py

```diff
@@ -320,27 +474,170 @@ def _add_missing_columns(self):
         self._data["longitude"] = None
         self._data["original_record_id"] = None
 
+    def _search_tokens_diff_order(self, areas, second):
```

Comment:
To improve the code readability, rename the argument "second" to "appended_text" and update all argument references.